contrib/enum_metrics/config.ini

```diff
@@ -0,0 +1,19 @@
+[localhost]
+cassandra_nodes = localhost
+es_nodes = localhost
+
```

Comment:
This file seems rax specific.  We try not to check such files in to the open BF repo.

Instead this may be a separate PR for bf-chef.  Pull all but localhost out for now.